ui/raidboss/raidboss_config.js

```diff
@@ -984,6 +988,22 @@ const templateOptions = {
     }
   },
   options: [
+    {
+      id: 'Coverage',
+      name: {
+        en: 'Supported content (latest version)',
+      },
+      type: 'html',
+      html: {
+        // TODO: it'd be nice if OverlayPlugin could open links on the system outside of ACT.
+        en: makeLink('https://quisquous.github.io/cactbot/util/coverage/coverage.html?lang=en'),
```

Comment:
I think it would be better to show link text as "Open link in new window" instead of an URL?

Reply:
The problem is that when you open a link in ACT, there is no url bar.  I thought it was better to show the url in case you wanted to open it in a browser or copy it or know where the link was.

Reply:
Oh, that make sense.
So, only if OverlayPlugin supports "Open in system browser" would solve the problem?
I can recall that you can run shell command like "start <url>" to invoke system browser, is it possible to be built in cactbot/OverlayPlugin?

Reply:
I'm not sure about what the best way to invoke the system browser is.  But, I do think that approach makes the most sense to me.  Add an overlay handler to open a system browser to OverlayPlugin and then switch cactbot to use that.